<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeMainWindow</class>
 <widget class="QMainWindow" name="HomeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备管理系统 · 首页</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#HomeWindow { background: #f6f8fb; }
QFrame#side_nav { 
	background-color: rgb(216, 235, 255);
}
QLabel#avatarLabel { 
	background:#ffffff; border-radius:28px; border:1px solid rgba(0,0,0,0.06); 
	padding: 5.5px
}

QToolButton#btnDevices, QToolButton#btnOrders,
QToolButton#btnThanks,
QToolButton#btnSettings {
    border:none; background:transparent; color:#4b5563; border-radius:10px;
	font-size:20px;
	padding-left:15px;
}
QToolButton#btnDevices:hover, QToolButton#btnOrders:hover,
QToolButton#btnThanks:hover, 
QToolButton#btnSettings:hover {
    background:rgba(37,99,235,0.08); color:#1f2937;
}
QToolButton#btnDevices:checked, QToolButton#btnOrders:checked,
QToolButton#btnThanks:checked
{
    background:rgba(37,99,235,0.15); color:#1e40af;
}

QFrame#appbar { 
	
	background-color: rgb(255, 255, 255);
}
QLabel#titleLabel { color:#111827; font-size:18px; font-weight:600; }
QLineEdit#searchEdit { background:#f3f4f6; border:1px solid rgbQWidget#centralwidget QFrame#side_nav QToolButton{padding-left:200px;}a(0,0,0,0.06); border-radius:14px; padding:8px 12px; }
QLineEdit#searchEdit:focus { border:1px solid #60a5fa; outline:3px solid rgba(96,165,250,0.25); }
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="side_nav">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="avatarLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>56</width>
             <height>56</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>56</height>
            </size>
           </property>
           <property name="toolTip">
            <string>未登录</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="pixmap">
            <pixmap resource="../image/UiTool.qrc">:/home/man.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_side">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnDevices">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>设备管理</string>
         </property>
         <property name="text">
          <string>设备状态</string>
         </property>
         <property name="icon">
          <iconset resource="../image/UiTool.qrc">
           <normaloff>:/home/geer-black.svg</normaloff>:/home/geer-black.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnOrders">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolTip">
          <string>工单管理</string>
         </property>
         <property name="text">
          <string>工单管理</string>
         </property>
         <property name="icon">
          <iconset resource="../image/UiTool.qrc">
           <normaloff>:/home/ticket-black.svg</normaloff>:/home/ticket-black.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="side_spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnThanks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="toolTip">
          <string>工单管理</string>
         </property>
         <property name="text">
          <string>致谢</string>
         </property>
         <property name="icon">
          <iconset resource="../image/UiTool.qrc">
           <normaloff>:/home/flower-color.svg</normaloff>:/home/flower-color.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>56</width>
           <height>56</height>
          </size>
         </property>
         <property name="toolTip">
          <string>设置</string>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
         <property name="icon">
          <iconset resource="../image/UiTool.qrc">
           <normaloff>:/home/setGeer-color.svg</normaloff>:/home/setGeer-color.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="appbar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>56</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <layout class="QHBoxLayout" name="appbarLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>个人主页</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="title_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="searchEdit">
            <property name="placeholderText">
             <string>搜索…</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnNotif">
            <property name="toolTip">
             <string>通知</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnProfile">
            <property name="toolTip">
             <string>我的</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackWidget">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../image/UiTool.qrc"/>
 </resources>
 <connections/>
</ui>
